_sass/blocks/_revenues_forms.scss

```diff
@@ -4,34 +4,34 @@
 //
 // Styleguide blocks.revenues_forms
 
-.revenues-section-forms {
-  background-color: $white;
-  padding-top: $base-padding-large;
-  padding-bottom: $base-padding-large;
+.revenues_page-forms {
+  padding-top: $base-padding;
+  padding-bottom: $base-padding;
 
-  h2 {
+  h1 {
     color: $light-black;
+    font-weight: $weight-light;
   }
 
   h3 {
     @include heading(h1);
-    border-bottom: 4px solid #cccccc;
+    border-bottom: $regular-border-size solid $dark-gray;
     padding-bottom: 5px;
     margin-bottom: 8px;
   }
 
   @include respond-to(small-up) {
-    .revenues-section-forms-options > div {
+    .revenues_page-forms_options > div {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `revenues_page-forms_options` should be written in lowercase with hyphens